msteams-platform/tabs/how-to/create-channel-group-tab.md

```diff
@@ -1028,11 +1028,9 @@ You've completed the tutorial to build a tab app with Blazor.
 
 > [!NOTE]
 >
-> * Migrating your configurable tab to static tab is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
-> * Migrating your configurable tab to static tab is available only in classic Teams client and isn't available in the [new Teams client](~/resources/teams-updates.md). 
-> * To migrate your configurable tab to static tab, use the app manifest v1.16 or later.
+> To migrate your configurable tab to static tab, use the app manifest v1.16 or later.
 
-Static tab capability is extended to support group chat, channels, and meetings. You can update your existing configurable tab to static tab and add different scopes to the static tab. 
+Static tab capability is extended to support group chat, channels, and meetings. In static tabs, the configuration dialog is optional, enabling users to pin the tab instantly. You have the option to add a configuration dialog to the tab at any point after it's pinned, as the `setConfig` APIs allows you to modify the `contentUrl`. You can update your existing configurable tab to a static tab and add different scopes to the static tab.
```

Comment:
This is not entirely true. This is a more accurate statement:
> You have the option to ~~add a configuration dialog to the tab at any point~~ **configure your tab** after it's pinned, as the `setConfig` APIs allows you to modify the `contentUrl` **which can useful to change the landing page if needed.**

Reply:
Updated